### ISSUE TITLE
Environment variables

### ISSUE THREAD

**Issue by bobrocke** [chuyik/brackets-snippets#50]
You mention adding these as one of your To Do items. So I'm creating this issue to say I could really use that feature - access to the clipboard contents, for sure.

**Comment by chuyik** [comment]
Yes this should be a cool feature, great to see somebody wants it.

**Comment by Yolo-Jerome** [comment]
Would this probably include something like "${parentfolder of actual open file}"?
If yes I would completely like that, too!!
If no .. I would open a new suggestion ;)
